src/index.js

```diff
@@ -51,13 +53,16 @@ const ping = (options, shouldMute) => {
     // Fires all requests
     .then(response => {
       log(`Loading details of the latest ${options.limit} top stories`);
-
-      return fetchTopStoriesDetails(response, options.limit);
+      return fetchTopStoriesDetails(response);
     })
     // Returns a formatted array with the response request
     .then(response => {
       return response.map(item => item.body);
     })
+    // Sort the reponse if requested
```

Comment:
@erickbelfy Maybe change this to "Sort the response if `options.latest`. "requested" resembles to HTTP calls which is not the case and might cause confusion.